spec/integration/tariff_synchronizer/rollback_cds_spec.rb

```diff
@@ -1,5 +1,5 @@
-RSpec.describe TariffSynchronizer, '.rollback_cds' do
-  subject(:rollback_cds) { described_class.rollback_cds('2021-01-01') }
+RSpec.describe CdsSynchronizer, '.rollback' do
```

Comment:
These filenames aren't quite right.

We'll want to rename these to be `spec/integration/cds_synchronizer/rollback_spec.rb`